app/assets/stylesheets/observations/uploader.scss

```diff
@@ -1119,3 +1119,11 @@ $tickmark: 6px;
 .new-time {
   padding-left: 5px;
 }
+
+.time-shifter-buttons {
+  margin-top: 15px;
+
+  button {
+    margin-right: 15px
+  }
+}
```

Comment:
Instead of this stuff, let's add the [`btn-toolbar`](https://getbootstrap.com/docs/3.4/components/#btn-groups-toolbar) class to the `time-shifter-buttons` div to get conventional bootstrap spacing.